browser/src/control/Control.JSDialog.js

```diff
@@ -639,7 +649,7 @@ L.Control.JSDialog = L.Control.extend({
 		instance.isDocumentAreaPopup = instance.popupParent === '_POPOVER_' && instance.posx !== undefined && instance.posy !== undefined;
 		instance.isPopup = instance.isModalPopUp || instance.isDocumentAreaPopup || instance.isSnackbar;
 		instance.containerParent = instance.isDocumentAreaPopup ? document.getElementById('document-container'): document.body;
-		instance.isAutofilter = instance.isDocumentAreaPopup && this.map._docLayer.isCalc();
+		instance.isAutofilter = instance.isDocumentAreaPopup && this.map._docLayer.isCalc() && instance.id !== 'autoFillPopup';
```

Comment:
here add `instance.isAutoFillPopup`, and also use it in `isAutofilter` to not duplicate detection

Reply:
done